operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/dependent/workflow/KubernetesResourceDeletedCondition.java

```diff
@@ -4,14 +4,6 @@
 import io.javaoperatorsdk.operator.api.reconciler.Context;
 import io.javaoperatorsdk.operator.api.reconciler.dependent.DependentResource;
 
-/**
```

Comment:
We should update this documentation accordingly, not just remove it.

Reply:
I removed since, from the code is very trivial, so self descriptive. Not sure if this needs to be explicitly documented.

Reply:
kk will add back docs.